osp-deploy/cinder/powerflex/README.md

```diff
@@ -157,8 +155,24 @@ san_password = powerflex_password
 
 ### Install SDC
 
-Install the Storage Data Client (SDC) on all nodes after deploying the overcloud
+Install the PowerFlex Storage Data Client (SDC) on all nodes after deploying the overcloud.
 
 ### Test the configured Backend
-Finally create a volume-type and test if you can successfully create and attach volumes of that type.
+Finally, create a PowerFlex volume type and test if you can successfully create and attach volumes of that type.
 
+Run the following command to check whether the Cinder service is started. 
+```
+[stack@rhosp-undercloud ~]$ source ~/overcloudrc
+(overcloud) [stack@rhosp-undercloud ~]$ cinder service-list
```

Comment:
All cinder subsequent commands should be using the openstack python client not cinder.
**openstack volume service list** would be the equivalent

Reply:
got it. will use - openstack volume service list